.travis.yml

```diff
@@ -55,6 +55,10 @@ install:
   rm -rf "$RUSTUP_HOME"toolchains/*/share
 before_script:
 - ./tools/setup.py
+- |-
+  # Start sccache, then throw away the S3 access key.
```

Comment:
Can you move this command to be a YAML comment. IE above the "- |-" so that in the log we can see the commands being run